packages/wdio/src/wdio.ts

```diff
@@ -38,45 +38,64 @@ export async function loadAxe(driver: BrowserObject): Promise<void> {
 /**
  * Load and run axe in given WDIO instance and return the accessibility violations found.
  */
-export async function runAxe(driver: BrowserObject, rules: A11yConfig = recommended): Promise<axe.Result[]> {
+export async function runAxe(
+    driver: BrowserObject,
+    context = 'html',
```

Comment:
I think the types for this param should be more specific. Right now it's defined as a string essentially, but it looks like the intersection of what `$` from wdio accepts and a `axe.ElementContext`. 

The type probably should be an actual DOM node or a CSS selector string.

Reply:
Actually, based on the doc of `assertAccessible` and the examples it looks like this only accepts a string as a selector. Do we want to support passing in a real DOM element?

Reply:
i have been thinking about that ..
wondering if we can add `assertAccessible` as a custom command on the `browser` (for full page a11y checking) and element objects with a `registerSa11yAPIs` helper method (similar to the one in jest). That way the APIs would be available directly on the browser/element without having to pass them in as a param?

https://webdriver.io/docs/customcommands.html

Reply:
Yeah adding to the `browser` API is another option. I typically only see those defined at the project level not automatically by a 3rd party tool like sa11y. But if that's what other tools are doing then I think it's worth exploring for us as well.